finetune/util/tokenization.py

```diff
@@ -0,0 +1,40 @@
+import unicodedata
+
+
+WEIRD_SPM_CHAR = "▁"
+WEIRD_TEXT = """This is a basic test of tokenizers but with some fun things at the end. 
```

Comment:
Can we move this constant to a test file?

Reply:
yep!